ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "oauth2"
-version = "2.12.1"
+version = "2.13.0"
```

Comment:
I guess we also need to update the distribution version in L10?

Reply:
Fixed with [31f4d9a](https://github.com/ballerina-platform/module-ballerina-oauth2/pull/1330/commits/31f4d9aeda3e6e10d3278371a970503acc1db2b6)